so/README.txt

```diff
@@ -0,0 +1,24 @@
+This dataset is derived from a dump of StackOverflow posts downloaded on June 10th 2016 from
```

Comment:
The `so` track seems to be *very* similar to the `nested` track. Is there a chance we can reuse / amend that track or do we need a separate one?

Reply:
It is based on the same source, but is unlike the other track not nested and does contain the question/answer text. I therefore think it will be difficult to merge them.